luaver

```diff
@@ -65,6 +65,31 @@ __luaver_exec_command()
     fi
 }
 
+# Return the correct make utility for this OS
+__luaver_get_make_tool()
+{
+    # Care for make tool compatibility
+    # If OS is *BSD, require gmake
+    if [[ $OSTYPE =~ linux ]]
+    then
+        echo "make"
+    elif [[ $OSTYPE =~ bsd ]]
+    then
+        echo "gmake"
+    fi
```

Comment:
Simply checking if `gmake` exists else falls back to `make` is good for portability, I think.
```bash
gmake -v > /dev/null 2>&1 && echo "gmake" || echo "make"
```